mc2mc/internal/loader/helper.go

```diff
@@ -0,0 +1,25 @@
+package loader
+
+import (
+	"strings"
+)
+
+func SeparateHeadersAndQuery(query string) (string, string) {
```

Comment:
can you also call this function to this part? https://github.com/goto/transformers/blob/bbb25fcf2000f99d7aee5e971a1dba19c7276739/mc2mc/internal/client/client.go#L95
we also need to reconstruct header and query here